pages.md

```diff
@@ -6,47 +6,31 @@ Note: When building links to module pages in module code, make sure to use the `
 
 
 #### Left-Hand Menu Project Links
-Links to pages can be configured to appear in REDCap's left-hand menu by adding them to `config.json`. Links configured under the `project` section (as shown below) will be visible by default for users with design rights, on all project pages, on projects where the module is enabled.  This behavior can be modified via the `redcap_module_link_check_display()` hook (see the [method documentation](methods/README.md) for details).  See [the config.json docs](config.md) for details on link configuration options.
+Links to pages can be configured to appear in REDCap's left-hand menu by adding them to `config.json`.  These can include module pages or links to outside websites.  Links configured under the `control-center` section (shown below) will appear only in REDCap's **Control Center**. Links configured under the `project` section (shown below) will be visible by default for users with design rights, on all project pages, on projects where the module is enabled.  Display behavior/permission and link details (name, url, etc.) can be modified on the fly via the `redcap_module_link_check_display()` hook (see the [method documentation](methods/README.md#em-hooks) for details).  See [the config.json docs](config.md) for details on link configuration options.
```

Comment:
Maybe highlight the restriction to design rights users more:

`... will be visible by default **only for users with design rights**, on all project pages, on projects ...`

Reply:
Done!